<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectorClass</class>
 <widget class="QDialog" name="ConnectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAddress"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDisconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Send data:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leIntervalMillisecond">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbHello">
       <property name="text">
        <string>Hello</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="teSendData"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Recieve data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="lwSendData"/>
     </item>
     <item>
      <widget class="QListWidget" name="lwRecvData"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Connector.qrc"/>
 </resources>
 <connections/>
</ui>
